### DISCUSSION TITLE
Auto fetch indexers from prowlarr

### DISCUSSION THREAD

**Discussion by LeaderbotX400** [cross-seed/cross-seed#744] [Ideas]
The prowlarr API makes it quite easy to do this and the code looks something like this.

```javascript
async () => {
    const apiKey = ""
    const rawData = await fetch(`http://prowlarr:9696/api/v1/indexer?apikey=${apiKey}`);
    const jsonData = await rawData.json();

    return jsonData.map((i) => `http://prowlarr:9696/${i.id}/api?apikey=${apiKey}`);
},
```
[API Reference](https://prowlarr.com/docs/api/#/Indexer/get_api_v1_indexer)

My only question then, is would this even be possible to do in the configuration?

**Comment by zakkarry** [comment]
Previous relevant discussions worth mentioning here are #474 #466 and #691
